fhir-database-utils/src/main/java/org/linuxforhealth/fhir/database/utils/common/DataDefinitionUtil.java

```diff
@@ -21,7 +21,7 @@
  * Handles common syntax for generating DDL
  */
 public class DataDefinitionUtil {
-    private static final String NAME_PATTERN_RGX = "[a-zA-Z_]\\w*$";
+    private static final String NAME_PATTERN_RGX = "[a-zA-Z_][-\\w]*$";
```

Comment:
today i [learned](https://stackoverflow.com/questions/3697202/including-a-hyphen-in-a-regex-character-bracket) that you can use an unescaped `-` to match within a character class if its the first (or last) char... i always escape these :-)

Reply:
I always just use [regex101](https://regex101.com/) for this 😄